src/Invictus.Testing.LogicApps/LogicAppAuthentication.cs

```diff
@@ -91,5 +105,13 @@ private static async Task<LogicManagementClient> AuthenticateLogicAppsManagement
                 SubscriptionId = subscriptionId
             };
         }
+
+        private static async Task<LogicManagementClient> AuthenticateLogicAppsManagementAsync(string subscriptionId, string accessToken)
```

Comment:
Let's refactor `Task<LogicManagementClient> AuthenticateLogicAppsManagementAsync(string subscriptionId, string tenantId, string clientId, string clientSecret)` so that it calls this method instead

Reply:
should be ok now